### ISSUE TITLE
"\\" code colouring broken

### ISSUE THREAD

**Issue by Extra-Mental** [Rusketh/ExpAdv2#263]
I made an EA2 that can request google images, but since there's no http encode functions, I had to make my own, along the way I discovered that "\\" beaks the colouring of anything underneath. It needs to be surrounded by a large amount of code in order for it to happen.

I think it may have something to do with \\ and \".

The code still compiles fine, but its difficult to to know what you are coding when it keeps flickering.

the table mine is in is:
table Encode = {
        " " = "%20",
        "-" = "%2D",
        "." = "%2E",
        "<" = "%3C",
        ">" = "%3E",
        "^" = "%5E",
        "_" = "%5F",
        "`" = "%60",
        "{" = "%7B",
        "|" = "%7C",
        "}" = "%7D",
        "~" = "%7E",
        "!" = "%21",
        "#" = "%23",
        "$" = "%24",
        "&" = "%26",
        "(" = "%28",
        ")" = "%29",
        "*" = "%2A",
        "+" = "%2B",
        "," = "%2C",
        "/" = "%2F",
        ":" = "%3A",
        ";" = "%3B",
        "=" = "%3D",
        "?" = "%3F",
        "@" = "%40",
        "[" = "%5B",
        "]" = "%5D",
        "\\" = "%5C",

    }

![untitled-1](https://cloud.githubusercontent.com/assets/13044464/8928444/d8267482-3561-11e5-9dde-096b302610d0.png)

Id rather not post all of my code

**Comment by ArmyArmy** [comment]
Does the code validate and work? If it does only the color is problem, if not the \\ is broken or not implemented.

**Comment by Extra-Mental** [comment]
Yes the code validates and compiles fine, also throughout the main post, there were supposed to be double back slashes, I guess github cancels them out
